docs/get-started.md

```diff
@@ -41,3 +41,27 @@ Using the url, branch, and deploy key, `wksctl` will clone the repo and create t
 These `--git` arguments are then used to set up and configure [flux](https://www.weave.works/oss/flux/) to automate cluster management via Git aka [GitOps](https://www.weave.works/technologies/gitops/)
 
 We will rely on the user installing [fluxctl](https://docs.fluxcd.io/en/latest/references/fluxctl.html#installing-fluxctl) to interact with flux directly.  `wksctl` does not replicate this functionality.
+
+### wksctl apply 
+A complete description of the apply command
```

Comment:
This section needs to be enclosed in:
\```console
\<text here>
\```
to be formatted correctly... otherwise, Markdown turns it into a single text paragraph.

Reply:
Almost... :-) The header "### wksctl apply" shouldn't be within the "console" section. You can see how it will look by clicking on the "..." at the upper right of the PR and selecting "View file".